docs/lesson/lesson.yaml

```diff
@@ -122,63 +122,6 @@ paths:
                 status: 404
                 code: DOCUMENT_NOT_FOUND
                 message: Lesson with the specified id was not found.
-    get:
```

Comment:
why did you removed it?

Reply:
Review the full file. We have a duplicate of the get method

Reply:
Besides that, on the develop branch we have an error based on this mistake

Reply:
fixed.